vertical-pod-autoscaler/deploy/vpa-rbac.yaml

```diff
@@ -291,6 +291,7 @@ rules:
       - delete
       - get
       - list
+      - patch
```

Comment:
Was just chatting with someone about this and there is a possible vulnerability here... If the admission-controller is somehow compromised, then the attacker now has a way to modify the webhook to do whatever they want.

Wonder if we should run this by SIG-Security to get their take?

Reply:
That exists already, right? Since it can create and delete webhooks, all it needs to do is create a new malicious one.

I think that's good reason why we should move to a different method of providing the web hook to users, such as a Helm chart or something equivalent.

Reply:
Oh, and just to be clear. I'm not saying that since it's broken already, it's Ok to make it worse.
If this is a concert with sig-security and it is a deal breaker, then I think we should figure out a better way to provide a "production deployment" experience as a replacement of this PR

Reply:
So for this, is there anything actionable on my end? If we don't want users to use this rbac, there should probably be some documentation about the whole `./hack/vpa-up.sh` deployment method in production, but I don't know if that's something for this PR.

Reply:
No nothing actionable, we need to put together some kind of documentation.